packages/expo-cli/src/appleApi/authenticate.ts

```diff
@@ -84,19 +85,19 @@ function _getAppleIdFromParams({ appleId, appleIdPassword }: Options): AppleCred
   }
 
   // none of the apple id params were set, assume user has no intention of passing it in
-  if (!appleId) {
+  if (!passedAppleId) {
     return null;
   }
 
   // partial apple id params were set, assume user has intention of passing it in
-  if (!(appleId && passedAppleIdPassword)) {
+  if (!(passedAppleId && passedAppleIdPassword)) {
```

Comment:
passedAppleIdPassword can only be defined if passedAppleId is defined

Reply:
I noticed that. Didn't want to make this patch more complicated than need be, but I can clean this up if you'd like.